info-daemn/Makefile

```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=info-daemon
+#PKG_VERSION:=1.0.0
+PKG_VERSION:=HEAD
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=$(TOPDIR)/../../packages/info-daemon
```

Comment:
The source URL should be remote i.e. https://github.com/CreatorDev/ci40-info-daemon.git

Developers should set the source link locally for development purpose